### DISCUSSION TITLE
nRF Connect SDK missing other board modules

### DISCUSSION THREAD

**Discussion by ShahinHaque97-CT** [zephyrproject-rtos/zephyr#57255] [Q&A] [answered]
Hello, 

The nRF Connect SDK, although it is used for Nordic IC, it runs on Zephyr RTOS. I would like to use this SDK for other boards too however there is missing files for the other boards. The nRF Connect SDK contains the board files in Zephyr/boards, however some files are missing for example, lets say the iMX-RT1050 is to be used.

The board file is present in C:\ncs\v2.3.0\zephyr\boards\arm\mimxrt1050_evk 

However C:\ncs\v2.3.0\modules\hal is missing the nxp folder.

How would I go about updating the modules?

**Comment by carlescufi** [comment]
NCS is not designed to be used with non-Nordic boards. In fact the Nordic 5-Clause license that most of its code uses would potentially prohibit it.

Can you share some info as to why you'd like to use NCS on an NXP board?

**Reply by ShahinHaque97-CT** [reply]
Just looking into the NFC reader using the ST25R3911B, Which I have running on the Nordic MCUs, ideally I would like to port this over but can be re-written if needed.

**Reply by nordicjm** [reply, marked as answer]
You would need to create your own implementation without any of the source files.
